.prettierrc

```diff
@@ -2,7 +2,7 @@
   "singleQuote": true,
   "trailingComma": "all",
   "tabWidth": 2,
```

Comment:
Can we have `tabWidth: 4`? I feel like 2 makes it a bit difficult to identify the indentations

Reply:
I'm going to say that this is something that could be discussed, but my feeling is that 4 space width doesn't work in this repo since we often have much greater indentation levels compared to say our python stuff (for example, `CameraRecording.tsx`). Either way probably something for a future PR if we decide to go with it. Only reason I modified the config here was because it was actually incorrect.